modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/StandardHeader.scala

```diff
@@ -381,6 +381,12 @@ class StandardHeader(
       val windowsCount = buildInt32(SeqAction(timingWindows.length), "NUMREQTW")
       sendKeywords(id, inst, hs, windowsCount :: windows)
     }
+    
+    def decodeGuide(v: StandardGuideOptions.Value): String = v match {
+      case StandardGuideOptions.Value.park   => "parked"
+      case StandardGuideOptions.Value.guide  => "guiding"
+      case StandardGuideOptions.Value.freeze => "frozen"
+    }
```

Comment:
So I guess the `_.toString` method doesn't match the what needs to be sent. How is it different? Why not fix it upstream?